tests/test_adapter_composition.py

```diff
@@ -118,6 +118,7 @@ def test_parallel_inference_with_heads(self):
 
             with self.subTest(model_class=model_class.__name__):
                 inputs = {}
+                inputs["attention_mask"] = torch.ones((2, 128))
```

Comment:
can we have a different attention mask for the different inputs? this would verify whether the attention masks are repeated correctly 

With the current test case we could not detect a difference between the two types of repetitions because a==b
(a,b) -> (a,a,b,b)
(a,b) -> (a,b,a,b)

Reply:
Updated to use a random attention mask for each input.